README.md

```diff
@@ -30,6 +30,44 @@ Now, in your application's HTML file, given they are within the scope of your mo
 with an optional (style) input to specify the color, size, or other properties of the main tree container.
 
 
+## Link locally with zlux-editor for local development      
+
+### Part I- How to link
+
+#### Add npm symlink 
+Note: @zlux/file-explorer is package name in package.json
+```
+cd zlux-file-explorer
+npm link 
+```
+#### Replace dependency in zlux-editor
+```
+cd zlux-editor/webClient
+npm link @zlux/file-explorer
+```
+
+### Part II - Back to normal, how to delink
+
+#### Remove Dependency link
+```
+cd zlux-editor/webClient
+npm unlink
+```
+
+#### Delete npm symlink
+
+```
+cd zlux-file-explorer
+npm uninstall
+```
+
+### node-sass vendor folder not found
```

Comment:
encountered this error when deleted node modules and trying `npm run build`, kind of troubleshooting tip while building locally